gnomad/assessment/validity_checks.py

```diff
@@ -857,12 +863,54 @@ def vcf_field_check(
     return True
 
 
+def compare_global_and_row_annot_lengths(
```

Comment:
Maybe add an option to compare all rows with a default of False. Just to make it clear it's only comparing the first row to the globals and with support to compare all if wanted.

Reply:
Maybe `check_global_and_row_annot_lengths`? or `confirm_global_and_row_annot_lengths`